### ISSUE TITLE
Application is installed while brew had error.

### ISSUE THREAD

**Issue by Kiehls** [Homebrew/brew#3165]
- [V] Confirmed this is a problem with running a `brew` command and not `brew install`ing or the post-install behaviour of one or more formulae? If it's a formulae-specific problem please file this issue at the relevant tap e.g. for Homebrew/homebrew-core https://github.com/Homebrew/homebrew-core/issues/new
- [V] Ran `brew update` and retried your prior step?
- [V] Ran `brew doctor`, fixed all issues and retried your prior step?
- [V] Ran `brew config` and `brew doctor` and included their output with your issue?

To help us debug your issue please explain:
- What you were trying to do (and why)
Trying to install "SmoothMouse" application through brew.

- What happened (include command output)
Installation was stopped and get some errors.
```
Failed command:
/usr/bin/sudo -E -- /usr/sbin/installer -pkg /usr/local/Caskroom/smoothmouse/1.0.10/SmoothMouse.pkg -target /
```

```
Standard Output of failed command:
installer: Package name is SmoothMouse
installer: Installing at base path /
installer: The install failed (설치 프로그램에서 오류가 발생하여 설치에 실패했습니다. 소프트웨어 제조업체에 지원을 문의하십시오.) - Error is Korean which means "Installation is failed due to installation program error. Ask Software Manufacture to support"
```
- What you expected to happen
Just expected not install the application but, I found that this 'SmoothMouse' application was appeared in my 'System Preferences'. 
Also I tried to find the app in 'Finder' but, there wasn't.

- Step-by-step reproduction instructions (by running `brew` commands)

> $ brew cask search smooth
> $ brew cask install smoothmouse

# Problems
Suggesting the problem that I encountered.

In short, I wanted to install "smoothmouse" with brew.
But, brew stoped with some errors. However, In my System Preference, 'SmoothMouse' was installed. In addition, Launchpad has no 'SmoothMouse' icon.

**Comment by ilovezfs** [comment]
Please report this to https://github.com/caskroom/homebrew-cask.
